packages/lesswrong/components/recentDiscussion/RecentDiscussionTag.tsx

```diff
@@ -115,9 +148,17 @@ const RecentDiscussionTag = ({ tag, refetch = () => {}, comments, expandAllThrea
   
   return <div className={classes.root}>
     <div className={classes.tag}>
-      <Link to={isSubforum ? tagGetSubforumUrl(tag): tagGetDiscussionUrl(tag)} className={classes.title}>
-        {tag.name}{isSubforum && " Subforum"}
-      </Link>
+      {isSubforum ? <div className={classes.subforumTitleRow}>
+        <div className={classes.subforumIcon}>
+          <TopTagIcon tag={tag} />
+        </div>
+        <div className={classes.subforumTitleText}>
+          <Link to={tagGetSubforumUrl(tag)} className={classes.subforumTitle}>{tag.name}</Link>
+          <div className={classes.subforumSubtitle}>subforum discussion</div>
```

Comment:
I think I slightly prefer this capitalised, happy to defer though

<img width="866" alt="Screenshot 2022-12-21 at 13 20 30" src="https://user-images.githubusercontent.com/77623106/208914886-9c27c418-f3fd-453a-abd0-59ddf1de224b.png">